src/MicrosoftTeams.ts

```diff
@@ -49,6 +49,176 @@ namespace microsoftTeams {
         args?: any[]; // tslint:disable-line:no-any The args here are a passthrough from OnMessage where we do receive any[]
     }
 
+     /**
+     * Namespace to send message to SDK from native apps.
+     */
+    export namespace native {
+        export function postMessage(msg: MessageEvent): void {
```

Comment:
postMessage is a way of communication from native client to SDK. 
You have rightly pointed, that this may confuse developers. I have added a comment to address this confusion.